config/canonical.json

```diff
@@ -31382,6 +31382,8 @@
     "count": 152,
     "tags": {
       "brand": "Tesco Metro",
+      "brand:wikidata": "Q487494",
```

Comment:
I also think that we should use Q57551648 instead of the generic Tesco entry...

Reply:
right